index.html

```diff
@@ -5,17 +5,17 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Personal Portfolio</title>
-    <link rel="stylesheet" href="/main.css">
+    <link rel="stylesheet" href="main.css">
     <style> @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap'); </style>
 </head>
 <body>
-    <header>
+    <header class="header">
         <section class="navSec">
             <div class="logo">
                 <h3>welcome</h3>
```

Comment:
[Optional]

- [ ] Please consider changing the `h3` tag to an `a` tag since this is the logo of the website, doing that will allow users to click on the logo to go to the top of the page 💯